### ISSUE TITLE
TorrentLeech Add Categories to Search Solution

### ISSUE THREAD

**Issue by PrivatePasta** [ronggang/PT-Plugin-Plus#1002]
TorrentLeech search is very messy and unless you limit the search to specific categories, we get a bunch of results that have nothing to do with our initial search. For this reason, I've just disabled search on TL cus it's so annoying. TorrentLeech already has categories you can select when searching. My suggestion is that you give us the option to select the categories we'd like to search with in the Search Solution. Right now we can only select "All".

**Comment by ted423** [comment]
The reason for the confusion of tl search is most likely because it searches for content profiles and filenames in torrents

And the maintainers didn't spend much time on it

**Comment by PrivatePasta** [comment]
Yeah, it's unfortunate. Would be cool if we could select categories in the search solution, maybe I can actually use search with TL then.

**Comment by ted423** [comment]
Please wait for the next https://github.com/ronggang/PT-Plugin-Plus/projects/2

**Comment by ted423** [comment]
https://www.torrentleech.org/torrents/browse/list/categories/26,32/query/Strange%20Sex

看起来tl是支持json的
故整体搜索可能需要更新

tl support json ，html search need change to use json
